.github/workflows/check.yml

```diff
@@ -0,0 +1,68 @@
+name: Flatpak CI
+
+on: [push]
```

Comment:
Do you not want `pull_request` also?

Reply:
At least on the YAML end, [it should work as expected](https://yaml-online-parser.appspot.com/?yaml=-+%7C%0A++a+%5C%0A++b&type=json).

Reply:
So, i added both, and then it started *two* builds....

Reply:
So, i think we just want push, that will be enough. Otherwise it will trigger a CI each time someone comments or something.